ch5-mkdir/makedir.cpp

```diff
@@ -0,0 +1,18 @@
+#include "makedir.h"
+
+#include <cerrno>
+#include <system_error>
+
+#include <sys/stat.h>
+
+void makeDir(const std::string& path)
```

Comment:
Can it be `string_view`?

Reply:
`mkdir()` requires a null-terminated string.

Reply:
Sure! Yeah, this is a tricky pitfall. Thanks.